sdk/textanalytics/Azure.AI.TextAnalytics/tests/Infrastructure/RetryOnInternalServerErrorAttribute.cs

```diff
@@ -34,20 +34,23 @@ private static bool ShouldRetry(TestExecutionContext context)
 
             // A known transient issue where the server returns a successful 200 OK response status but reports an
             // internal server error in the response body for one or more text analysis tasks.
+            // https://github.com/Azure/azure-sdk-for-net/issues/35677
```

Comment:
I recommend a `// TODO: ` prefix or even `// BUGBUG: `. I don't recall for the latter, but I know the former actually shows up in Visual Studio's task view so they are easy to track down. Either also adds a more obvious sign "look here!".

(BUGBUG is old school)

Reply:
Fixed!